core/datainput.gms

```diff
@@ -714,10 +698,14 @@ $offdelim
 $Onlisting
 
 
-*CG* setting wind off capacity factor to be the same as onshore here (later adjusting it in vm_capFac)
-$ifthen.WindOff %cm_wind_offshore% == "1"
-f_cf(ttot,regi,"windoff") = f_cf(ttot,regi,"wind");
-$endif.WindOff
+*CG* setting wind offshore capacity factor to be the same as onshore here (later adjusting it in vm_capFac)
+*** windoffshore-todo 
+f_cf(ttot,regi,"windon") $ (f_cf(ttot,regi,"windon") eq 0) = f_cf(ttot,regi,"wind");
```

Comment:
what is this needed for? 
is it just for the period when you want to use the new REMIND but still have old input data with "wind" in it?

Reply:
yes, i did not update mrremind yet, so we need an intermediate version compatible with the two names (:

Reply:
do you mean (f_cf(ttot,regi,"windon") **ne** 0)? is f_cf ever 0..?

Reply:
The meaning is:
If mrremind data does not contain a value for windon (because it is still using wind), then define the variable for windon by copying the value of wind (and discard the wind variable because we won't need it in remind)

I added a comment before each occurrence of this temporary trick, hopefully it avoids confusion:
> *** allow input data with either "wind" or "windon" until mrremind is updated